frontend/src/services/user-service.js

```diff
@@ -63,11 +63,23 @@ export class UserService {
             });
     }
 
-    searchUserByName(name, limit = 10) {
-        return this.http.get(`users?limit=${limit}&name=${name}&fields=firstName,lastName,email`)
+    getQueryParams(name, type, limit) {
+        return `limit=${limit}&name=${name}&fields=firstName,lastName,email&userType=${_.isUndefined(type) ? '' : type}`;
```

Comment:
if `type` is undefined, `userType` should no longer be placed in the queryParams